src/DI/SecurityExtension.php

```diff
@@ -21,7 +21,7 @@ class SecurityExtension extends CompilerExtension
 	public function loadConfiguration(): void
 	{
 		$builder = $this->getContainerBuilder();
-		$config = $this->validateConfig($this->defaults);
+		$config = $this->defaults;
```

Comment:
Method is deprecated so you just deleted call of it? Config is not validated anymore and is not even used, just default values...